src/static/client/app/signup/signup.component.scss

```diff
@@ -1,110 +1,129 @@
 * {
-    box-sizing: border-box;
-    margin: 0;
-    padding: 0;
-  }
-  
-  body, html {
-    height: 100%;
-    font-family: Arial, sans-serif;
-  }
-  
-  .signup {
-    background: linear-gradient(to right, #1632aa, #B30D2C);
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body,
+html {
+  height: 100%;
+  font-family: Arial, sans-serif;
```

Comment:
Can this be changed to Poppins google font?

Reply:
Yes, I can certainly change the font to Poppins. I'll make the necessary changes to import the Poppins font from Google Fonts and apply it to the body and html tags. Thank you for the suggestion.

Reply:
The font is still Arial?